azure-devops/azext_devops/dev/team/_format.py

```diff
@@ -59,9 +59,54 @@ def _transform_service_endpoint_row(row):
     return table_row
 
 
+def transform_teams_table_output(result):
+    table_output = []
+    for item in sorted(result, key=_get_team_key):
+        table_output.append(_transform_team_row(item))
+    return table_output
+
+
+def transform_team_table_output(result):
+    table_output = [_transform_team_row(result)]
+    return table_output
+
+
+def _transform_team_row(row):
+    table_row = OrderedDict()
+    table_row['ID'] = row['id']
```

Comment:
I think we should have constant variables for these.

Reply:
Opened 
https://github.com/Microsoft/vsts-cli/issues/330 to track this

Reply:
Suggestion: Shouldn't they be defined in the same file? Since they are coming from json which is specific to command outputs in this file. Otherwise as u said tracking casing and all will be an issue.